src/UKHO.S100PermitService.API/Program.cs

```diff
@@ -95,12 +100,37 @@ private static void ConfigureServices(WebApplicationBuilder builder)
             {
                 options.Headers.Add(Constants.XCorrelationIdHeaderKey);
             });
+
             var options = new ApplicationInsightsServiceOptions { ConnectionString = configuration.GetValue<string>("ApplicationInsights:ConnectionString") };
-            builder.Services.AddApplicationInsightsTelemetry();
+            builder.Services.AddApplicationInsightsTelemetry(options);
 
             builder.Services.AddControllers();
             builder.Services.AddEndpointsApiExplorer();
+
             builder.Services.Configure<EventHubLoggingConfiguration>(builder.Configuration.GetSection("EventHubLoggingConfiguration"));
+
+            var azureAdConfiguration = new AzureADConfiguration();
+            builder.Configuration.Bind("AzureADConfiguration", azureAdConfiguration);
```

Comment:
use AzureAd instead of AzureAD whenever applicable.


```suggestion
            var azureAdConfiguration = builder.Configuration.GetSection("AzureAdConfiguration").Get<AzureAdConfiguration>();
```

Reply:
This name is added as per the ReSharper's suggestion, Should I still change it to AzureAD?